frontend/src/setupTests.ts

```diff
@@ -15,19 +15,40 @@ if (!window.ResizeObserver) {
 
 process.env.TZ = 'UTC';
 
+const errorsToIgnore = [
+    'Warning: An update to %s inside a test was not wrapped in act',
+    "Failed to create chart: can't acquire context from the given item",
+];
+
+const warningsToIgnore = [
+    '[MSW] Found a redundant usage of query parameters in the request handler URL for',
+    'MUI: You have provided an out-of-range value',
+];
+
+const logsToIgnore = ['An exception was caught and handled.'];
```

Comment:
We *can* probably put them all in a single list, when I think about it. It's not like you're gonna get the same message as a log and as a warning, is it?